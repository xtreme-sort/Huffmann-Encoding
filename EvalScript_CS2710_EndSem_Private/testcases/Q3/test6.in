GenerateTreeEnhanced
29
4 1
L 3
K 7
F 8
Y 8
n 10
q 10
0 11
W 11
w 12
X 13
2 19
T 20
c 24
V 25
o 30
y 32
7 33
P 33
v 33
6 37
i 38
p 38
x 44
U 45
3 46
S 47
5 50
l 50
Quit
