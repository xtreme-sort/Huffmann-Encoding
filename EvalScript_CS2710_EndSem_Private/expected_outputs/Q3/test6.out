((((v,6),(i,p)),(((2,T),((n,q),(0,W))),(x,U))),(((3,S),(((((4,L),K),w),c),5)),((l,(V,(X,(F,Y)))),((o,y),(7,P)))));
